recipes/pandas_plink/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pandas_plink" %}
+{% set version = "1.0.12" %}
+{% set sha256 = "4b9f770cbfef9f0f088e3336c74344d5f55856f245009b2f4ea5ad3dc2c49254" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or not py27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cffi >=1.7
+    - libffi
+    - pytest-runner
+  run:
+    - python
+    - cffi >=1.7
+    - numpy
+    - pandas
+    - tqdm
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pandas_plink
+  commands:
+    - python -c "import pandas_plink; pandas_plink.test()"
+
+about:
+  home: http://github.com/Horta/pandas-plink
```

Comment:
Also following http://github.com/Horta/pandas-plink suggests that this should use `-` instead of `_`. We should determine whether it should be `_` or `-` before proceeding to the feedstock.